extension/content_script.js

```diff
@@ -490,6 +489,7 @@ In the INACTIVE state, the widgets are not visible, and no event handlers are at
 function oneTimeSetup() {
 	let readableDomEls = window.parseDocument();
 	doc = new Doc(readableDomEls);
+	doc.updateWordModel(readableDomEls);
```

Comment:
Renamed to "updateWordModel", since updates TF, IDF, and initiates setKeyWord(), which determines keywords. Previously it was "calcTotalWords"